src/graphics/context.rs

```diff
@@ -113,10 +113,14 @@ impl GraphicsContext {
             })
         };
 
+        let mut base_limits = wgpu::Limits::default();
         if conf.backend == Backend::All {
+            base_limits = wgpu::Limits::downlevel_webgl2_defaults();
+
             match Self::new_from_instance(
                 game_id,
                 new_instance(wgpu::Backends::PRIMARY),
+                base_limits.clone(),
```

Comment:
```suggestion
                wgpu::Limits::downlevel_default(),
```

If we request a primary backend, default limits are guaranteed to be available (`downlevel_default` whould be enough for us though).